makefile/.konveyor/ansible/setup_env.mk

```diff
@@ -0,0 +1,13 @@
+### Important, the LANG_DIR must match one from Makefile
+LOCAL_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
```

Comment:
Nice ! :) we can remove the LANG_DIR comment above.

Reply:
Done , and expanded to include python dir as well. So it is all the same.